docs/providers/aws/cli-reference/install.md

```diff
@@ -32,4 +32,12 @@ serverless install --url https://github.com/some/service
 serverless install --url https://github.com/johndoe/authentication
 ```
 
-This example will download the .zip file of the `authentication` service from GitHub, create a new directory with the name `authentication` in the current working directory and unzips the files in this directory.
\ No newline at end of file
+This example will download the .zip file of the `authentication` service from GitHub, create a new directory with the name `authentication` in the current working directory and unzips the files in this directory.
+
+### Installing a service from directories in a GitHub URL
+
+```bash
+serverless install --url https://github.com/johndoe/authentication/tree/master/todo/someauth
```

Comment:
could you use https://github.com/serverless/examples/tree/master/rest-api-with-dynamodb in example?